assets/app/views/deployments.html

```diff
@@ -9,14 +9,210 @@
         <div>
           <em>{{emptyMessage}}</em>
         </div>
-      </div>      
-      <div style="margin-bottom: 10px;" ng-repeat="deployment in deployments">
-        <h3>{{deployment | annotation:'deploymentConfig'}} <span class="small">({{deployment.metadata.name}})</span></h3>
-        <div>Created: <relative-timestamp timestamp="deployment.metadata.creationTimestamp"></relative-timestamp></div>
-        <div>Status: {{deployment | annotation:'deploymentStatus'}}</div>
-        <div>Version: {{deployment | annotation:'deploymentVersion'}}</div>
-        <div>Replicas: {{deployment.spec.replicas}}</div>
-        <pod-template ng-init="podTemplate = deployment.spec.template"></pod-template>
+      </div> 
+
+      <div class="tile" ng-repeat="(deploymentConfigName, deploymentConfig) in deploymentConfigs track by (deploymentConfig | uid)">
+        <div> 
+          <h2>{{deploymentConfigName}}</h2>
+          <dl class="dl-horizontal left indent">
+            <div>
+                <dt>Created:</dt>
+                <dd><relative-timestamp timestamp="deploymentConfig.metadata.creationTimestamp"></relative-timestamp></dd>
+            </div>
+            <div>
+                <dt>Labels:</dt><dd ng-if="!deploymentConfig.metadata.labels">none</dd>
+                <dd ng-repeat="(labelKey, labelValue) in deploymentConfig.metadata.labels">{{labelKey}}={{labelValue}}<span ng-show="!$last">, </span></dd>
+            </div>
+            <div>
+                <dt>Latest Version:</dt>
+                <dd>{{deploymentConfig.latestVersion}}</dd>
+            </div>
+            <div ng-if="deploymentConfig.template.strategy.type">
+                <dt>Strategy:</dt>
+                <dd>{{deploymentConfig.template.strategy.type}}</dd>
+            </div>
+
+            <div>
+              <h3>Template:</h3>
+              <dl class="dl-horizontal left indent">
+                <dt>Selectors:</dt><dd ng-if="!deploymentConfig.template.controllerTemplate.replicaSelector">none</dd>
+                <dd ng-repeat="(selectorLabel, selectorValue) in deploymentConfig.template.controllerTemplate.replicaSelector">{{selectorLabel}}={{selectorValue}}<span ng-show="!$last">, </span></dd>
+                <dt>Replicas:</dt>
+                <dd>
+                  {{deploymentConfig.template.controllerTemplate.replicas}}
```

Comment:
ah, osapi v1beta1... hrmm... don't really want to add something we're going to have to immediately rework when we switch to osapi v1beta3